CODEOWNERS

```diff
@@ -10,6 +10,9 @@
 # File patterns work mostly like .gitignore. Try to keep this file
```

Comment:
Off-topic sorry but can you please drop the "javascript" comment above and replace it with a link to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners?

It looks like copy/pasta from there.

This feature is too complex, it's badly missing a test script like
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/scripts/get_maintainer.pl

Reply:
Ack, added @marc-hb